addons/l10n_ch/models/res_bank.py

```diff
@@ -271,7 +271,7 @@ def _is_qr_iban(self):
         different.
         """
         self.ensure_one()
-        return self.acc_type == 'iban' \
+        return self.sanitized_acc_number.startswith('CH') and self.acc_type == 'iban' \
```

Comment:
It seems ok but plz fix the indentation!
```
return condition1 and condition2 \
           and condition3
``` is bad...
It should be
```
return condition1 and condition2 and condition3
```
 or 
```
return condition1\
          and condition2\
          and condition3
```
@oco-odoo Does it look ok for you too? :)